schedule/templates/create_selection.haml

```diff
@@ -8,5 +8,8 @@
 
   :javascript
     $('document').ready(function(){
-      $('#id_courses').select2();
-    });
\ No newline at end of file
+        $('#id_courses').selectize({
```

Comment:
I blindly trust you for this

Reply:
It looks like this 
![screenshot from 2017-09-28 23-39-05](https://user-images.githubusercontent.com/85785/30991683-5a5dd3ae-a4a6-11e7-8ae1-9f5ebfb7b548.png)
It's also faster and handles multiline when the input is long